bootstrap/shoppingmall_seed.py

```
@@ -852,241 +764,34 @@ def seed_backoffice_data(db, state: SeedState) -> None:
-            info("ShoppingMall 데이터베이스 초기화 완료")
-        else:
-            info("ShoppingMall 데이터베이스 상태 확인 완료")
-        return 0
-    except Exception as exc:
-        error(str(exc))
+        _log("ShoppingMall 코어/백오피스 시드 시작")
+        state = seed_core_data(db)
+        seed_backoffice_data(db, state)
+        db.commit()
+        _log("ShoppingMall 시드 완료")
         return 1
```

Ruff (0.15.12)

[warning] Consider moving this statement to an `else` block

(TRY300)